src/api/icon.api.js

```diff
@@ -131,20 +131,20 @@ export class DrawingIcon {
     }
 
     /**
-     * Generate an icon URL from its template. If no iconSize is given, medium scale will be
+     * Generate an icon URL from its template. If no iconScale is given, default scale 1 will be
      * applied. If no iconColor is given, red will be applied (if applicable, as non-colorable icons
      * will not have {r}, {g}, {b} part of their template URL)
      *
-     * @param {FeatureStyleSize} iconSize The size (or scale) to use for this icon's URL
      * @param {FeatureStyleColor} iconColor The color to use for this icon's URL
+     * @param {Number} iconScale The scale to use for this icon's URL
      * @returns {String} A full URL to this icon on the service-icons backend
      */
-    generateURL(iconSize = MEDIUM, iconColor = RED) {
+    generateURL(iconColor = RED, iconScale = 1) {
```

Comment:
why not using a constant anymore? (SMALL, MEDIUM, etc...)

Reply:
Because the icon scale of MEDIUM, ... is the open layer scale and here I want to be sure to get the icon at its scale 1 by default and we always takes icon from backend at the same scale and let openlayer do the scaling.